### DISCUSSION TITLE
why can't we change the size when in music mode?

### DISCUSSION THREAD

**Discussion by RedemptionC** [iina/iina#4489] [General]
I mean, it'd be better if the interface can be bigger/maximized
<img width="412" alt="image" src="https://github.com/iina/iina/assets/17563754/49171636-8bed-462b-8db0-e2e498774316">
Also, when in video mode, why can't we change the size of the chapters/playlist panel?
it'd be more convenient if the whole file title can be showed

**Comment by low-batt** [comment]
The current answer to this:
> why can't we change the size when in music mode?

Is because testing the changes in pull request #4381 that address issue #3484 and issue #3764 revealed some problems. So that PR is currently in the `Draft` state awaiting further development.

This is considered a desirable enhancement. Of course there is lots and lots of work for developers to do.

**Reply by RedemptionC** [reply]
I see. and thanks to Mac's hover text feature, it's less demanding, for me.
